docs/mkdocs.yml

```diff
@@ -1,10 +1,120 @@
-site_name: Catalyst CI Documentation
+# cspell: words Diagramsnet, pymdownx, linenums, inlinehilite, superfences,
+# cspell: words tasklist, betterem, materialx, twemoji, smartsymbols,
+# cspell: words kroki, excalidraw, glightbox, cips, pygments, arithmatex
+
+# Project Information
+site_name: Project Catalyst - Catalyst CI Documentation
+site_url: https://input-output-hk.github.io/catalyst-ci
+site_author: The Project Catalyst Team
+copyright: (c) 2023 Input Output
+
+# Repository
+repo_name: input-output-hk/catalyst-ci
+repo_url: https://github.com/input-output-hk/catalyst-ci
+
+# The Documentation src to build.
 docs_dir: src
+
 theme:
   name: material
+  custom_dir: overrides
+  palette:
+    # Palette toggle for light mode
+    - scheme: default
+      media: "(prefers-color-scheme: light)"
+      toggle:
+        icon: material/brightness-7
+        name: Switch to dark mode
+
+    # Palette toggle for dark mode
+    - scheme: slate
+      media: "(prefers-color-scheme: dark)"
+      toggle:
+        icon: material/brightness-4
+        name: Switch to light mode
+
   features:
-    - navigation.indexes
+    - navigation.instant
+    - navigation.instant.progress
     - navigation.tracking
```

Comment:
I'm not sure which setting is doing this, but the navigation sidebar now includes sections, instead of showing a ToC on the left side. I think I really preferred it the other way, because some documents have a lot of headings, and it completely blows up the navigation bar and makes it really hard to figure out where you are.

Reply:
Will check it.
You are right to nitpick this because I am wanting the cat-ci mkdocs.yml to be our preferred starting point for all docs.

Reply:
New navigation style applied for review.